packages/playground/src/components/node_selector/TfManualNodeSelector.vue

```diff
@@ -99,11 +105,15 @@ export default {
 
     // reset node to mark form as invalid
     const placeholderNode = ref<NodeInfo>();
+    const filters = computed(() => normalizeNodeFilters(props.filters));
```

Comment:
should we remove the computed and pass the  `normalizeNodeFilters(props.filters));` to the function directly?